### PR TITLE
[xcode14.1] Bump to Xcode 14.1 Beta 3

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

**Comment by dalexsoto** [comment]
I can't Approve my own PR but 👍 thank you @mandel-macaque for taking it over

File: src/BackgroundAssets/BACompat.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
+//
+// BACompar.cs: Compatibility functions
```
Comment (tj-devel709) [inline src/BackgroundAssets/BACompat.cs:2]:
```suggestion
// BACompat.cs: Compatibility functions
```

File: tests/cecil-tests/AttributeTest.cs [hunk 2]
```diff
@@ -272,20 +272,24 @@ static bool IgnoreElementsThatDoNotExistInThatAssembly (string member)
...
+				"AuthenticationServices.ASAuthorizationSecurityKeyPublicKeyCredentialRegistration",
+				"AuthenticationServices.ASAuthorizationSecurityKeyPublicKeyCredentialDescriptor",
+				"GameController.GCRacingWheel.get_HandlerQueue",
+				"GameController.GCRacingWheel.set_HandlerQueue",
+				"GameController.GCRacingWheel.get_VendorName",
+				"GameController.GCRacingWheel.get_ProductCategory",
+				"GameController.GCRacingWheel.get_PhysicalInputProfile",
+			};
+
+			if (inlineProtocols.Any ((p) => member.StartsWith (p))) {
```
Comment (tj-devel709) [inline tests/cecil-tests/AttributeTest.cs:289]: Nice ^

**Review by tj-devel709 — APPROVED** [review]
LGTM just a tiny typo in a comment

**Comment by rolfbjarne** [comment]
Test failures are unrelated (https://github.com/xamarin/maccore/issues/2450).

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**